app/src/main/java/org/p2p/wallet/solend/interactor/SolendWithdrawInteractor.kt

```diff
@@ -0,0 +1,72 @@
+package org.p2p.wallet.solend.interactor
+
+import org.p2p.solanaj.core.Account
+import org.p2p.solanaj.core.FeeAmount
+import org.p2p.solanaj.model.types.Encoding
+import org.p2p.solanaj.rpc.RpcSolanaRepository
+import org.p2p.wallet.feerelayer.interactor.FeeRelayerAccountInteractor
+import org.p2p.wallet.feerelayer.program.FeeRelayerProgram
+import org.p2p.wallet.infrastructure.network.provider.TokenKeyProvider
+import org.p2p.wallet.relay.RelayRepository
+import org.p2p.wallet.rpc.repository.blockhash.RpcBlockhashRepository
+import org.p2p.wallet.solend.model.SolendDepositToken
+import org.p2p.wallet.solend.repository.SolendRepository
+import org.p2p.wallet.utils.toBase58Instance
+import java.math.BigInteger
+
+class SolendWithdrawInteractor(
+    private val rpcSolanaRepository: RpcSolanaRepository,
+    private val rpcBlockhashRepository: RpcBlockhashRepository,
+    private val solendRepository: SolendRepository,
+    private val relayRepository: RelayRepository,
+    private val feeRelayerAccountInteractor: FeeRelayerAccountInteractor,
+    private val tokenKeyProvider: TokenKeyProvider
+) {
+
+    suspend fun withdraw(token: SolendDepositToken, amountInLamports: BigInteger): String {
```

Comment:
withdraw -> withdrawToken

Reply:
I don't really want to rename the obvious name.
You  can see that this interactor is about Solend withdraw and you see that there is a token in params. 

So why should I duplicate a token in the method name? 
One of my proposal is to make the naming as shorter as possible